projects/packages/stats-admin/src/class-rest-controller.php

```diff
@@ -695,6 +706,49 @@ public function get_site_plan_usage( $req ) {
 		);
 	}
 
+	/**
+	 * Post user feedback for Jetpack Stats.
+	 *
+	 * @param WP_REST_Request $req The request object.
+	 *
+	 * @return array
+	 */
+	public function post_user_feedback( $req ) {
+		$current_user = wp_get_current_user();
```

Comment:
Login is already checked in `can_user_view_general_stats_callback` I believe.

Reply:
Ah, indeed. So we can just use the current user. 👍🏼

Reply:
Addressed in [877a769](https://github.com/Automattic/jetpack/pull/39287/commits/877a76926fd054e5ea42ce3b1538c2213233fe06).